docs/marimo/mlir_ir.py

```diff
@@ -1,6 +1,6 @@
 import marimo
 
-__generated_with = "0.10.17"
+__generated_with = "0.11.0"
```

Comment:
I have not opened this with marimo, but will assume this change doesn't break stuff

Reply:
yeah it's just an automatic version bump on save

Reply:
I'm not sure if there's an easy way to do this in the CI when updating marimo with dependabot